src/scripts/generate-heading-ids.js

```diff
@@ -57,10 +59,22 @@ const addHeaderID = (line, slugger, write = false) => {
     .replace(/\{#[^}]+\}/, "")
     .trim()
   const headingLevel = line.slice(0, line.indexOf(" "))
+
+  curMaxLevel =
+    headingLevel.length > curMaxLevel ? headingLevel.length : curMaxLevel
+
+  const headingLevelArrayIndex = headingLevel.length - 1
+  if (
+    curLevel[headingLevelArrayIndex] == undefined ||
```

Comment:
May be able to simplify this statement by just checking if it's a truthy value or not, i.e. 
```
  if (!curLevel[headingLevelArrayIndex]) {
    curLevel[headingLevelArrayIndex] = 0
  }
```

Reply:
Of course it's better 👍🏽